### PR TITLE
Service contract changes done

### PR DESCRIPTION
**Pull request by RakeshKumarVienna**

### CODE REVIEW

File: ModelLibrary/Process/CreateContractInvoice.cs [hunk 1]
```diff
@@ -328,5 +343,30 @@ private void GenerateInvoice(VAdvantage.Model.X_C_ContractSchedule contSchedule)
...
+        /// Created Date: 01/Apr/2021
+        /// Purpose: Get DocType
+        /// </summary>
+        private void SetDocType()
+        {
+            _C_DocType_ID = 0;
+            // When ContractType is Accounts Receivable
+            if (cont.GetContractType() == X_C_Contract.CONTRACTTYPE_AccountsReceivable)
+            {
+                sql.Append(MRole.GetDefault(GetCtx()).AddAccessSQL("SELECT MIN(C_DOCTYPE_ID) FROM C_DocType WHERE DOCBASETYPE='ARI' AND ISACTIVE ='Y'", "C_DocType", true, true));
```
Comment (Manjot-Vienna) [inline ModelLibrary/Process/CreateContractInvoice.cs:357]: Use function from MDocType to get document type based on Document Base Type

**Review by Manjot-Vienna — CHANGES_REQUESTED** [review]
PLease do the changes

**Review by Manjot-Vienna — APPROVED** [review]
OK

File: ModelLibrary/Process/CreateContractInvoice.cs (hunk 1 above)
Comment (RakeshKumarVienna) [reply ModelLibrary/Process/CreateContractInvoice.cs]: Changes Done

**Review by Bharat-Vienna — APPROVED** [review]
OK

**Pull request merged**